js/binance.js

```diff
@@ -4971,24 +4971,21 @@ module.exports = class binance extends Exchange {
         };
         if (since !== undefined) {
             request['startTime'] = since;
-            const sinceString = since.toString ();
-            const limitString = Precise.stringSub (limit.toString (), '1');
```

Comment:
This endpoint actually returns one more than the limit for some reason

Before this was removed, it would do this
```
% binance fetchBorrowRateHistory USDT 1616284800000 5
2021-12-18T06:40:56.164Z
Node.js: v14.17.0
CCXT v1.64.15
binance.fetchBorrowRateHistory (USDT, 1616284800000, 50)
642 ms
currency |   rate |     timestamp |                 datetime
------------------------------------------------------------
    USDT | 0.0014 | 1620518400000 | 2021-05-09T00:00:00.000Z
    ...
    USDT | 0.0015 | 1616284800000 | 2021-03-21T00:00:00.000Z
50 objects
```
 
 But I can't test the new behavior, because apparently `this.min` is not a function

Reply:
It is supposed to be `Math.min`, my bad, fixed that.